### DISCUSSION TITLE
Copilot/Typescripts hides code completion in VS Code

### DISCUSSION THREAD

**Discussion by grapevinegizmos** [community/community#21040] [Copilot Conversations] [answered]
I've been finding it quite frustrating that once I installed copilot I'm finding it very difficult see code completion suggestions in the editor for things like members of a enum.  

Instead I get names, not in the enum, made up by copilot and I haven't been able to find any keyboard shortcuts to give me the code hinting about to show the existing enum choices.

**Comment by FahdSeddik** [comment, marked as answer]
Hey!
Try pressing Ctrl+Space. Hope that helps! Be sure to mark it as an answer if it does, so people would find it easier.

**Reply by grapevinegizmos** [reply]
Unfortunately That key combo pops a keyboard selection dialog on Macbook Pro (OS Monterey V 12.4)

**Reply by FahdSeddik** [reply]
If you open the keyboard shortcut pane (just press Command + K, then followed by Command+S), then search for trigger suggest, then you should find the keybind you would need to press.

**Reply by grapevinegizmos** [reply]
Thanks.  Ctrl+space is listed as short cut but I guess overridden somehow by OS.
But Command-I also listed as an alternative -- and that one worked.  So thanks!
